Lib/fontTools/feaLib/parser.py

```diff
@@ -1062,6 +1128,15 @@ def expect_float_(self):
         raise FeatureLibError("Expected a floating-point number",
                               self.cur_token_location_)
 
+    def expect_decipoint_(self):
+        if self.next_token_type_ == Lexer.FLOAT:
+            return self.expect_float_() * 10
```

Comment:
OK, I see what is going here. The value stored in the binary is a decipoint, but FontTools converts to/from float when writing/reading the value, so setting 10 here will result in 100 being stored in the binary table. I’ll fix the code.